doc/devel/development_setup.rst

```diff
@@ -4,12 +4,33 @@
 Setting up Matplotlib for development
 =====================================
 
+Retrieving the latest version of the code
+=========================================
+
+Matplotlib is hosted at https://github.com/matplotlib/matplotlib.git.
+
+You can retrieve the latest sources with the command (see
+:ref:`set-up-fork` for more details)::
+
+    git clone https://github.com/matplotlib/matplotlib.git
+
+This will place the sources in a directory :file:`matplotlib` below your
+current working directory.
+
+If you have the proper privileges, you can use ``git@`` instead of
+``https://``, which works through the ssh protocol and might be easier to use
+if you are using 2-factor authentication.
```

Comment:
```suggestion
```

No one should be pushing directly to the upstream fork (even if they can) as we do all work through PRs from individual forks.

Reply:
Let's put that change into #23588?

Reply:
Sounds good to me.